packages/react-native/ReactCommon/react/renderer/core/RawPropsParser.h

```diff
@@ -21,13 +21,14 @@ namespace facebook::react {
  * Specialized (to a particular type of Props) parser that provides the most
  * efficient access to `RawProps` content.
  */
-class RawPropsParser final {
```

Comment:
Let's move this to a separate PR so we can discuss it more.

Make these not final means we have to be more careful about API changes, as OSS users may have overrides - additionally, if we need to add virtual methods, this adds perf overhead.